src/common.hpp

```diff
@@ -17,6 +18,11 @@
 #ifndef PQREVISION
 #define PQREVISION "unknown"
 #endif
+
+#ifdef __APPLE__
+#define PLATFORM_ID "Darwin"
```

Comment:
Please set it to `Linux` in the `#else` part to make it work on Linux.

Reply:
Fixed